src/components/30-organisms/commercial-hero-banner/story.js

```diff
@@ -32,6 +32,7 @@ export const CommercialHeroBanner = () => {
 
   const wrapper = document.createElement('div');
   const template = html`
+    <div class="m-landingpage-keyvisual">
```

Comment:
@bozojovicic do we really need this? I don't think that class is part of the patterns lib code

Reply:
You are right, I removed it.